lib/pact_broker/client/cli/pact_commands.rb

```diff
@@ -8,13 +9,14 @@ class PactPublicationError < ::Thor::Error; end
 
       module PactCommands
         using PactBroker::Client::HashRefinements
+        using PactBroker::Client::StringRefinements
 
         def self.included(thor)
           thor.class_eval do
             desc 'publish PACT_DIRS_OR_FILES ...', "Publish pacts to a Pact Broker."
-            method_option :consumer_app_version, required: true, aliases: "-a", desc: "The consumer application version"
+            method_option :consumer_app_version, aliases: "-a", desc: "The consumer application version", default: ''
```

Comment:
Why default empty string, and not just null?

Reply:
reasons I cannot remember xD

Reply:
should I explicitly add in `default: nil`